metrics/gdc.csv

```diff
@@ -1,4 +1,5 @@
 Name,Labels,Description,Child,Initialize
+wmo_wis2_gdc_metrics_info,,"Diagnostic of GDC centre identifier, API URL, and GB subscribed to",-,true
```

Comment:
add labels: centre_id|instance|subscribed_to|url

Reply:
The above is supposed to be a "info" metric (https://prometheus.github.io/client_python/instrumenting/info/), which prometheus / OpenMetrics allows for.

Thinking more, if we consider this (in the WIS2 context) as "not a metric", I could simply add this as an endpoint somewhere else in GDC.  The question is then whether a given global service should only emit WIS2 metrics as a rule.